blackbox/docs/config/secure.rst

```diff
@@ -0,0 +1,73 @@
+.. highlight:: sh
+
+.. _conf-secure-settings:
+
+===============
+Secure Settings
+===============
+
+Some settings are sensitive as they might store passwords, access keys or 
+other similar information. This type of data needs more protection than just 
+filesystem permissions and even more, should not be available in plain text.
+
+For these instances, we provide a keystore and the ``crate-keystore`` tool to 
+manage the settings in the keystore.
+
+.. NOTE::
+
+    Changes to the keystore take affect only after restarting CrateDB
+
+.. NOTE::
+
+    The CrateDB keystore currently provides obfuscation, but in the future 
+    password protection will be added.
+
+.. NOTE::
+
+    Not all settings are read from the keystore. Consult the documentation for 
+    each setting to see if it is secure setting.
```

Comment:
- please explain what type of setting is read from the key store
- please say whether this is configurable or hard-wired
- please link to the correct doc

Reply:
> please explain what type of setting is read from the key store
But we open this page with that saying it's "sensitive configuration"

> please say whether this is configurable or hard-wired
> please link to the correct doc

I don't think we should list here all the secure configurations but rather as stated mark the configurations as secure where they are documented

Reply:
I agree. but presumably we can still identify what type of config is included?